docs/AutomationProtocols.md

```diff
@@ -0,0 +1,98 @@
+- Microsoft Edge -> [Edge Driver](https://developer.microsoft.com/en-us/microsoft-edge/tools/webdriver/)
+- Internet Explorter -> [InternetExplorerDriver](https://github.com/SeleniumHQ/selenium/wiki/InternetExplorerDriver)
+- Safari -> [SafariDriver](https://developer.apple.com/documentation/webkit/testing_with_webdriver_in_safari)
+
+For any kind of mobile automation you will need to install and set up [Appium](http://appium.io/) which will allow you to automate mobile (iOS/Android) or even desktop (Mac/Windows) application using the same WebdriverIO setup. There are also plenty of services that allow you to run your automation test in the cloud at high scale. Instead of having to setup all these drivers locally you can just talk to these services (e.g. [SauceLabs](https://saucelabs.com/)) in the cloud and inspect the results on their platform. The communication between test script and automation environment will look as folows:
+
+![WebDriver Setup](/img/webdriver.png)
+
+### Advantages
+
+- Official W3C web standard that is supported by all major browser
+- Simplified protocol that covers common user interaction
+- Support for mobile automation (and even native desktop apps)
+- Can be used locally as well as in the cloud through services like [SauceLabs](https://saucelabs.com/)
+
+### Disadvantages
+
+- Not designed for in depth browser analysis (e.g. tracing or intercepting network events)
+- limited set of automation capabilities (e.g. no support to throttle CPU or network)
+
+## DevTools Protocol
+
+The DevTools interface is a native browser interface that is usually being used to debug the browser from a remote application (e.g. [Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools/)). Next to its capabilities to inspect the browser in nearly all possible forms it can also be used to control it. While evey browser used to have its own internal DevTools interface that was not much expose to the user, more and more browser now adapt the [Chrome DevTools Protocol](https://chromedevtools.github.io/devtools-protocol/) that is used to either debug a web application using Chrome DevTools or control Chrome using tools like [Puppeteer](https://pptr.dev/). The communication happens without any proxy, directly to the browser using WebSockets:
+
+![DevTools Setup](/img/devtools.png)
+
+WebdriverIO allows you to use this automation technology as an alternative for WebDriver if you have special requirements to automate the browser. With the [`devtools`](https://www.npmjs.com/package/devtools) NPM package you can use the same commands that WebDriver provides which then can be used by WebdriverIO and the WDIO testrunner to run its useful commands on top of that protocol. It uses Puppeteer to under the hood and allows you to run a sequence of commands with Puppeteer if need to. To use DevTools as automation protocol you have to install the NPM package as follows:
```

Comment:
`WebdriverIO allows you to use this automation technology as an alternative for WebDriver if you have special requirements to automate the browser.` what is the reference to "this"? I think this line could be rewrote in a way it makes more sense to the reader.